### ISSUE TITLE
[0.2.0] hamlit cli render template exception: undefined method options

### ISSUE THREAD

**Issue by felixbuenemann** [k0kubun/hamlit#6]
```bash
gem install hamlit
echo '!!! 5' > foo.haml
hamlit render foo.haml
/Users/felix/.gem/ruby/2.2.0/gems/hamlit-0.2.0/lib/hamlit/engine.rb:29:in `block in create': undefined method `options' for Hamlit::HTML::Ugly:Class (NoMethodError)
	from /Users/felix/.gem/ruby/2.2.0/gems/hamlit-0.2.0/lib/hamlit/engine.rb:28:in `select'
	from /Users/felix/.gem/ruby/2.2.0/gems/hamlit-0.2.0/lib/hamlit/engine.rb:28:in `create'
	from /Users/felix/.gem/ruby/2.2.0/gems/hamlit-0.2.0/lib/hamlit/engine.rb:18:in `block in <class:Engine>'
	from /Users/felix/.gem/ruby/2.2.0/gems/temple-0.6.10/lib/temple/mixins/engine_dsl.rb:117:in `call'
	from /Users/felix/.gem/ruby/2.2.0/gems/temple-0.6.10/lib/temple/mixins/engine_dsl.rb:117:in `block in chain_proc_constructor'
	from /Users/felix/.gem/ruby/2.2.0/gems/temple-0.6.10/lib/temple/engine.rb:60:in `call'
	from /Users/felix/.gem/ruby/2.2.0/gems/temple-0.6.10/lib/temple/engine.rb:60:in `block in call_chain'
	from /Users/felix/.gem/ruby/2.2.0/gems/temple-0.6.10/lib/temple/engine.rb:60:in `map'
	from /Users/felix/.gem/ruby/2.2.0/gems/temple-0.6.10/lib/temple/engine.rb:60:in `call_chain'
	from /Users/felix/.gem/ruby/2.2.0/gems/temple-0.6.10/lib/temple/engine.rb:50:in `call'
	from /Users/felix/.gem/ruby/2.2.0/gems/hamlit-0.2.0/lib/hamlit/cli.rb:41:in `generate_code'
	from /Users/felix/.gem/ruby/2.2.0/gems/hamlit-0.2.0/lib/hamlit/cli.rb:10:in `render'
	from /Users/felix/.gem/ruby/2.2.0/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
	from /Users/felix/.gem/ruby/2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
	from /Users/felix/.gem/ruby/2.2.0/gems/thor-0.19.1/lib/thor.rb:359:in `dispatch'
	from /Users/felix/.gem/ruby/2.2.0/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
	from /Users/felix/.gem/ruby/2.2.0/gems/hamlit-0.2.0/bin/hamlit:6:in `<top (required)>'
	from /Users/felix/.gem/ruby/2.2.0/bin/hamlit:23:in `load'
	from /Users/felix/.gem/ruby/2.2.0/bin/hamlit:23:in `<main>'
	from /Users/felix/.gem/ruby/2.2.0/bin/ruby_executable_hooks:15:in `eval'
	from /Users/felix/.gem/ruby/2.2.0/bin/ruby_executable_hooks:15:in `<main>'
```

**Comment by k0kubun** [comment]
Thank you for reporting an issue. I got the same issue in a difference place.
It seems temple v0.6.10 does not have the method. The original idea of calling the method is stallen from slim. Because slim requires at least v0.7.3, I changed the hamlit's requirement to it too. 05229f6

Released as v0.3.0. Could you check it?
I could reproduce the issue and confirmed that it works with v0.7.3. Closing.

**Comment by felixbuenemann** [comment]
Thanks, works fine!
